resources/lrsql/config/test/default/webserver.edn

```diff
@@ -32,4 +32,6 @@
  :oidc-enable-local-admin   false
  :enable-clamav             false
  :clamav-host               "localhost"
- :clamav-port               3310}
+ :clamav-port               3310
+ :auth-by-cred-id           false ;may need to be set to true if lrsql behind proxy
```

Comment:
When adding config vars, make sure you don't forget to also add them to a) the production config and b) the documentation.

Reply:
also don't comment that, because like we'd have to explain EVERY opt by that logic.

Reply:
documented, added to prod conf, uncommented

Reply:
I don't see it in the documentation README for env vars

Reply:
OK I see it added